packages/avatar/src/Avatar/AvatarContents/AvatarContents.style.ts

```diff
@@ -0,0 +1,55 @@
+import { css } from '@leafygreen-ui/emotion';
+
+import {
+  avatarFontSizeMap,
+  avatarMultiCharacterFontSizeMap,
+} from '../Avatar.constants';
+import { AvatarProps, AvatarSize, AvatarStyleArgs } from '../Avatar.types';
+
+const SINGLE_CHAR_FONT_SCALE = 0.5; // Scalar for `sizeOverride` to font-size
```

Comment:
I hate to admit this, but I looked up what `scalar` means, and I don't fully understand the meaning. Can you add a little more to the comment?

Reply:
They come from visually testing sizes until it looked right :shrug: